src/js/iotjs.js

```diff
@@ -69,9 +69,6 @@ setInterval = _timeoutHandler.bind(this, 'setInterval');
 clearTimeout = _timeoutHandler.bind(this, 'clearTimeout');
 clearInterval = _timeoutHandler.bind(this, 'clearInterval');
 
-// Make sure that it is not accessible elsewhere
-delete _timeoutHandler;
```

Comment:
Yes, that seems like a good solution, I didn't really think it through honestly, just wanted to hotfix the issue, and leave the solution to the original committer.